ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -81,7 +81,7 @@ def execute( self ):
     return S_OK()
 
   @executeWithUserProxy
-  def __syncCSWithVOMS( self, vo ):
+  def __syncCSWithVOMS( self, vo, proxyUserName, proxyUserGroup ):
```

Comment:
Specify default values, to None if needed, as otherwise there is a high risk of failure.